### PR TITLE
Scaling fonts

### PR DESCRIPTION
**Pull request by peteryates**
### Trello card

https://trello.com/c/1As2jCCk/1492-improve-font-scaling-for-mobile-based-on-govuk-design-guide

### Context and changes

Add some basic font scaling to the styles, globally. This will allow us to fit
some more text on the screen on mobile.

| Page | Before | After |
| ------ | ------ | ------ |
| Home | Screenshot from 2021-04-13 13-25-21 | Screenshot from 2021-04-13 13-25-28 |
| Steps (top) | Screenshot from 2021-04-13 13-26-02 | Screenshot from 2021-04-13 13-25-56 |
| Steps (scrolled down a bit) | Screenshot from 2021-04-13 13-26-12 | Screenshot from 2021-04-13 13-26-28 |
| Ways to train | Screenshot from 2021-04-13 13-26-46| Screenshot from 2021-04-13 13-27-02 |
| CTA | Screenshot from 2021-04-13 13-27-16 | Screenshot from 2021-04-13 13-27-26 |
| Funding (table) | Screenshot from 2021-04-13 13-28-05 | Screenshot from 2021-04-13 13-28-11 |
| Events | Screenshot from 2021-04-13 13-31-33 | Screenshot from 2021-04-13 13-31-49 |

### CODE REVIEW

**Comment by MylesJarvis** [comment]
This looks like such an improvement. We chatted over Slack about this @peteryates but listing here.

Main issues appear to be around events in terms of:

1. Scaling of titles (e.g. TTT Events compared with its title on the tile)
2. Iconography scaling on event listings
3. Discover events and scaling, along with padding top and bottom

**Comment by peteryates** [comment]
## Event headings have been tweaked (in da70b51f1)

| Tablet | Mobile |
| Screenshot from 2021-04-13 14-00-41 | Screenshot from 2021-04-13 14-00-50 |

## Icons now scale too


https://user-images.githubusercontent.com/128088/114556559-befd5900-9c60-11eb-9640-032cea608ba9.mp4

**Comment by HettieS** [comment]
Had a play around with this as well, felt like on my iPhone 6s the font was maybe slightly too small - compared it with BBC News app which seemed to have slightly larger font. Have suggested we bump mobile up to 85%. Otherwise looks good.

File: app/webpacker/styles/font-sizes.scss [hunk 1]
```diff
@@ -1,29 +1,56 @@
...
+  //   "xsmall": powderblue,
+  //   "small": plum,
+  //   "medium": coral,
+  //   "large": lawngreen,
+  //   "xlarge" : fuchsia,
+  //   "xxlarge": mediumspringgreen);
+
+  // @each $name, $whatever in $desktop-sizes {
+  //   outline: 1px dashed map-get($debug-colours, $size);
+  // }
```
Comment (peteryates) [inline app/webpacker/styles/font-sizes.scss:55]: Left this in because it's super-useful to future people working out wtf is going on with our font sizes 😅

**Comment by MylesJarvis** [comment]
On Icon Scaling, is it possible to make it so that the text still sits beside the icon?

**Comment by peteryates** [comment]
It was more so the text isn't dwarfed by the icon really.

**Review by ethax-ross — DISMISSED** [review]
Code-wise this looks good and it works well. I think that the portrait mobile font size is slightly _too_ small though; I find the landscape font size about as small as I think is comfortable (this is on my 6S):

**Review by ethax-ross — APPROVED** [review]

**Comment by MylesJarvis** [comment]
> On Icon Scaling, is it possible to make it so that the text still sits beside the icon?

Uploading for info:
image (4)
image (7)

File: app/webpacker/styles/font-sizes.scss (hunk 1 above)
Comment (HettieS) [reply app/webpacker/styles/font-sizes.scss:55]: v nice debugging!

**Comment by peteryates** [comment]
We actually do that with CTAs currently, will create another PR to address any fallout from this one.

Screenshot from 2021-04-14 12-30-55

**Pull request merged**